app/helpers/service_helper/textual_summary.rb

```diff
@@ -94,7 +94,17 @@ def textual_parent_service
   end
 
   def textual_orchestration_stack
-    @record.try(:orchestration_stack)
+    ost = @record.try(:orchestration_stack)
+    if ost && !ost.id.present?
+      {
+        :label => _("Orchestration Stack"),
+        :image => "orchestration_stack",
```

Comment:
@lgalis the format of the images in textual summaries will change with https://github.com/ManageIQ/manageiq/pull/13089 .. so depending on which gets merged first, you may need to do `:image => "100/orchestration_stack.png"`